### DISCUSSION TITLE
Community Meeting Saturday, November 23, 2024

### DISCUSSION THREAD

**Discussion by wrashi** [GothenburgBitFactory/taskwarrior#3693] [Community Meetings]
Our bi-weekly community meeting happens Saturday, November 23!

Starting time is 1600 UTC. 
Here is a link where you can check out what that means in your timezone: [World Clock Meeting Times](https://www.timeanddate.com/worldclock/meetingtime.html?month=11&day=23&year=2024&p1=5&p2=3904&p3=3918&p4=3921&iv=0).
The meeting details are below.

Google Meet joining info
Video call link: https://meet.google.com/tns-suqq-iaj
Or dial: ‪(CA) +1 289-323-1423‬ PIN: ‪830 535 232‬#
More phone numbers: https://tel.meet/tns-suqq-iaj?pin=8726596308824

To get a sense of what the meeting is like, you can check out some of our latest meetings here:
[TaskWarrior Community Meetings YouTube Playlist](https://www.youtube.com/playlist?list=PLU-esvACzq9XBiGjDCQnMQigKrjlNBq2W).

**Comment by wrashi** [comment]
I'm delighted that Chad Phillips has agreed to demonstrate how he uses TaskWarrior at tomorrow's Community Meeting!
https://discord.com/channels/796949983734661191/796964492235767858/1309186625917423730

**Comment by wrashi** [comment]
The recording of this morning's meeting can be found at the link below.
Overview: Developer updates & Chad's demonstration of TaskWarrior and OneNote (see link above). 
My favourite takeaway: setting keys in Vit is _way_ more powerful that I had imagined!

https://youtu.be/j2VZrsjiD5g

**Reply by zilto** [reply]
I love how straightforward and useful that note trick is
